DEVELOPMENT.md

```diff
@@ -1,18 +1,63 @@
 # Development
 
-## Localizations
+## Installing and building
 
-### Implementation
+To get started working with this site, clone the repository and install dependencies by running ```yarn``` or ```yarn install```
+
+
+
+This project uses Gatsby, so to start a development server, run:
+
+```
+yarn develop
+```
+
+Alternatively you can run the built version by first running
+
+```
+yarn build
+```
+
+and then serving it by running
+
+```
+yarn start
+```
+
+When you're done, simply open the site on the specified localhost port and you're good to go!
+
+### Other commands
```

Comment:
I like these, maybe we should consider something like this for the other repos as well later on?